examples/demo/examples/start.rs

```diff
@@ -14,6 +14,6 @@ async fn main() -> loco_rs::Result<()> {
         port: boot_result.app_context.config.server.port,
         binding: boot_result.app_context.config.server.binding.to_string(),
     };
-    start::<App>(boot_result, serve_params).await?;
+    start::<App>(boot_result, serve_params, true).await?;
```

Comment:
i would invert the bool meaning
the regular action is that we show banner
the special action is that someone requests a no-banner
so i would name the flag `hide_banner` with default false

Reply:
Changed.